README.md

```diff
@@ -1 +1,254 @@
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">Bootstrap</a></li>
+    <li><a href="#">React</a></li>
+    <li><a href="#">Redux</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+#### Home page
+
+- When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
+- Along with the category name, you will display some numeric values per category that come from the API.
+- When the user clicks (or taps) on a category item, the application navigates to the details page.
+
+### Details page
+
+- In the details page, the webapp retrieves data from the API to show detailed data in the category.
+- When the user clicks on the "Back" button (<), the user navigates to the home page.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://metricswebapp.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- DESCRIPTION VIDEO LINK -->
+
+## 🚀 Description Video Link <a name="live-demo"></a>
+
+- [Description Video Link](https://www.loom.com/share/98e3385a8dee47eeb2014bbbdc039ca3)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1. Before you go ahead with this React tutorial, please make sure you have:
+
+- Basic understanding of HTML and CSS.
+- JavaScript fundamentals (object, array, conditionals etc).
+- Familiarity with JavaScript ES6 features (class syntax, arrow functions, object destructuring etc).
+
+2. [Node and npm](https://nodejs.org/en/download/) are installed. Here are the versions I'll be using while making this tutorial:
+
+   ```shell
+   $ node --version
+   v16.13.2
+
+   $ npm --version
+   8.1.2
+   ```
+
+   > Installing npm adds two commands to the system—`npm` and `npx`—both of which I'll be using while making this tutorial.
+
+3. [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) is installed. Here's the version I'll be using while making this tutorial:
+
+   ```shell
+   $ git --version
+   git version 2.29.1.windows.1
+   ```
+
+4. A [GitHub](https://github.com/signup) account. :octocat:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/developertariq/bookstore.git
+
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
+
+### Deployment
+
+You can deploy this project using:
```

Comment:
- Since the app is deployed, kindly state which platform you've used in deploying the app.

Reply:
Hi @zieeco I have done as suggested.